pages/index.tsx

```diff
@@ -1,76 +1,122 @@
+import { useState } from 'react'
+import { DragDropContext, DropResult } from 'react-beautiful-dnd'
 import type { NextPage } from 'next'
+import dynamic from 'next/dynamic'
 import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import { ColumnsWrapper } from '../components/columns-wrapper'
+import { initialData } from '../data/initalData'
+
+// TODO implement DropSpace
+// import { DropSpace } from '../components/drop-space'
+
+const Column = dynamic(() => import('../components/column/Column'), {
+  ssr: false,
+})
 
 const Home: NextPage = () => {
+  const [state, setState] = useState<KanbanState>(initialData)
+
+  const onDragEnd = (result: DropResult) => {
```

Comment:
Might be nice to move everything to a component so that the page only handles loading and updating the data and doesn't care about the UI.